### PR TITLE
Add `Filiph Siitam Sandström` to the list

### PR DESCRIPTION
**Pull request by filiphsps**

### CODE REVIEW

**Review by linusg — APPROVED** [review]
🇸🇪

File: people.json [hunk 1]
```diff
@@ -798,5 +798,14 @@
       "GitHub": "https://github.com/MacDue"
     },
     "contributor": true
+  },
+  {
+    "nick": "Filiph Siitam Sandström",
+    "coordinates": [58.699,  12.453],
```
Comment (linusg) [inline people.json:804]:
```suggestion
    "coordinates": [58.699, 12.453],
```

to make the linter happy

**Comment by linusg** [comment]
ping @filiphsps

**Comment by filiphsps** [comment]
> ping @filiphsps

Yep, Will fix ASAP. Been kneedeep in other work and those few moments when I’m free I haven’t really been anywhere close to a `git push -f` capable machine hehe.

**Comment by filiphsps** [comment]
Alright, @linusg we should be good to go now :^)

Sorry for the extreme delay.

File: people.json [hunk 2]
```diff
@@ -839,5 +839,16 @@
       "LinkedIn": "https://www.linkedin.com/in/bplaat/"
     },
     "contributor": true
+  },
+  {
+    "nick": "Filiph Siitam Sandström",
+    "coordinates": [58.699, 12.453],
+    "links": {
+      "Website": "https://www.sweetsideofsweden.com/",
+      "Company": "https://nordcom.io/",
```
Comment (linusg) [inline people.json:848]: Listing a company feels out of place for this list :)

**Review by linusg — CHANGES_REQUESTED** [review]

File: people.json (hunk 2 above)
Comment (filiphsps) [reply people.json]: Yeah, LinkedIn makes way more sense to list! ☺️

**Pull request merged**